persistence/sql/update/update.go

```diff
@@ -21,13 +23,14 @@ type Model interface {
 	GetNID() uuid.UUID
 }
 
-func Generic(ctx context.Context, c *pop.Connection, tracer trace.Tracer, v Model, columnNames ...string) error {
+func Generic(ctx context.Context, c *pop.Connection, tracer trace.Tracer, v Model, columnNames ...string) (err error) {
 	ctx, span := tracer.Start(ctx, "persistence.sql.update")
-	defer span.End()
+	defer otelx.End(span, &err)
 
 	quoter, ok := c.Dialect.(interface{ Quote(key string) string })
 	if !ok {
-		return errors.Errorf("store is not a quoter: %T", c.Store)
+		err = errors.Errorf("store is not a quoter: %T", c.Store)
+		return
```

Comment:
This is not needed. By naming the return value in the function parameter, Go automatically assigns that variable name to the return value. You don't need to set `err` explicitly. Please revert